src/seedu/addressbook/AddressBook.java

```diff
@@ -30,7 +30,6 @@
  * in a text file.
  **/
 public class AddressBook {
-
```

Comment:
Please revert all changes not relevant to the PR. This is to keep `git-blame` useful in this repo.